### PR TITLE
chore: remove nft feature flag

### PR DESCRIPTION
**Pull request by vm**
Now that it's ungated we can remove the feature flag fully.

### CODE REVIEW

File: src/components/WalletModal/index.tsx [hunk 1]
```diff
@@ -314,31 +309,6 @@ export default function WalletModal({
...
-          <ExternalLink href="https://uniswap.org/terms-of-service/">Terms of Service</ExternalLink> and consent to its{' '}
-          <ExternalLink href="https://uniswap.org/privacy-policy">Privacy Policy</ExternalLink>.
-        </Trans>
-      )
-      return nftFlagEnabled ? (
-        <AutoRow style={{ flexWrap: 'nowrap', padding: '4px 16px' }}>
-          <ThemedText.BodySecondary fontSize={16} lineHeight="24px">
-            {content}
-          </ThemedText.BodySecondary>
-        </AutoRow>
```
Comment (cbachmeier) [inline src/components/WalletModal/index.tsx:332]: I would appreciate @aballerr input here, but it seems like this section is still to be included

File: src/components/NavBar/SearchBarDropdown.tsx [hunk 2]
```diff
@@ -343,7 +337,7 @@ export const SearchBarDropdown = ({
   ])
 
   return (
-    <Box className={isPhase1 ? styles.searchBarDropdownNft : styles.searchBarDropdown}>
```
Comment (cbachmeier) [inline src/components/NavBar/SearchBarDropdown.tsx:346]: can you remove this no longer used style?

File: src/components/NavBar/SearchBar.tsx [hunk 3]
```diff
@@ -199,15 +190,13 @@ export const SearchBar = () => {
                 setSearchValue(event.target.value)
               }}
               onBlur={() => sendAnalyticsEvent(EventName.NAVBAR_SEARCH_EXITED, navbarSearchEventProperties)}
-              className={`${styles.searchBarInput} ${
-                showCenteredSearchContent ? styles.searchContentCentered : styles.searchContentLeftAlign
```
Comment (cbachmeier) [inline src/components/NavBar/SearchBar.tsx:203]: can you remove this style if it is no longer used

File: src/components/NavBar/SearchBar.tsx [hunk 4]
```diff
@@ -160,12 +151,12 @@ export const SearchBar = () => {
...
         width={{ sm: isOpen ? 'viewWidth' : 'auto', md: 'auto' }}
         ref={searchRef}
-        className={isPhase1 ? styles.searchBarContainerNft : styles.searchBarContainer}
+        className={styles.searchBarContainerNft}
         display={{ sm: isOpen ? 'inline-block' : 'none', xl: 'inline-block' }}
       >
         <Row
           className={clsx(
-            ` ${isPhase1 ? styles.nftSearchBar : styles.searchBar} ${!isOpen && !isMobile && magicalGradientOnHover} ${
+            ` ${styles.nftSearchBar} ${!isOpen && !isMobile && magicalGradientOnHover} ${
```
Comment (cbachmeier) [inline src/components/NavBar/SearchBar.tsx:159]: can you remove these no longer used styles?

**Review by cbachmeier — COMMENTED** [review]
I think there's one section of Web3Status that isn't supposed to be removed but other than that only minor non-blocking nits

File: src/components/WalletModal/index.tsx (hunk 1 above)
Comment (aballerr) [reply src/components/WalletModal/index.tsx:332]: I tracked down the original pr, it looks like Charlie is right and this should stay.  

https://github.com/Uniswap/interface/pull/4790

Comment (vm) [reply src/components/WalletModal/index.tsx:332]: will add back

File: src/components/NavBar/SearchBarDropdown.tsx (hunk 2 above)
Comment (vm) [reply src/components/NavBar/SearchBarDropdown.tsx:346]: yes

**Review by cbachmeier — APPROVED** [review]
LGTM

**Pull request merged**